### PR TITLE
Deprecate .NET Functions extension packages

### PR DESCRIPTION
**Pull request by scottaddie**
While taking inventory of Functions extension packages, I noticed these are marked as deprecated on nuget.org. Updating the CSV file accordingly. No migration guides are available.

Also, removing the entry for an AuthTokens package that doesn't exist. For context, see https://github.com/Azure/azure-sdk-for-net/issues/31717#issuecomment-1275149462.

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -573,24 +573,23 @@
...
 "Microsoft.Azure.WebJobs.Extensions.Http","3.2.0","","WebJobs Extensions - Http","WebJobs","https://github.com/Azure/azure-webjobs-sdk-extensions/tree/v3.0.2/src/WebJobs.Extensions.Http","NA","NA","client","False","","","active","","","","","",""
 "Microsoft.Azure.WebJobs.Extensions.Kafka","3.5.0","","WebJobs Extensions - Kafka","WebJobs","https://github.com/Azure/azure-functions-kafka-extension/tree/3.0.0/src/Microsoft.Azure.WebJobs.Extensions.Kafka","NA","NA","client","False","","","active","","","","","",""
-"Microsoft.Azure.WebJobs.Extensions.MicrosoftGraph","","1.0.0-beta6","WebJobs Extensions - Microsoft Graph","WebJobs","https://github.com/Azure/azure-functions-microsoftgraph-extension","NA","NA","client","False","","","","","","","","",""
+"Microsoft.Azure.WebJobs.Extensions.MicrosoftGraph","","1.0.0-beta6","WebJobs Extensions - Microsoft Graph","WebJobs","https://github.com/Azure/azure-functions-microsoftgraph-extension","NA","NA","client","False","","","deprecated","","","","","",""
```
Comment (JoshLove-msft) [inline _data/releases/latest/dotnet-packages.csv:592]: This package doesn't seem to exist on Nuget.

Comment (scottaddie) [reply _data/releases/latest/dotnet-packages.csv]: Removed it

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -573,24 +573,22 @@
...
 "Microsoft.Azure.WebJobs.ProjectTemplates","4.0.2288","","WebJobs - Project Templates","WebJobs","https://github.com/Azure/azure-functions-templates/tree/3.1.1582","NA","NA","client","False","","","active","","","","","",""
-"Microsoft.Azure.WebJobs.Script.WebHost","","1.0.0-beta3","WebJobs - Scrip Web Host","WebJobs","https://github.com/Azure/azure-functions-host/tree/dev/src/WebJobs.Script.WebHost","NA","NA","client","False","","","","","","","","",""
-"Microsoft.Azure.WebJobs.Script","","1.0.0-beta3","WebJobs - Script","WebJobs","https://github.com/Azure/azure-functions-host/tree/dev/src/WebJobs.Script","NA","NA","client","False","","","","","","","","",""
+"Microsoft.Azure.WebJobs.Script.WebHost","","1.0.0-beta3","WebJobs - Script Web Host","WebJobs","https://github.com/Azure/azure-functions-host/tree/dev/src/WebJobs.Script.WebHost","NA","NA","client","False","","","deprecated","","","","","",""
+"Microsoft.Azure.WebJobs.Script","","1.0.0-beta3","WebJobs - Script","WebJobs","https://github.com/Azure/azure-functions-host/tree/dev/src/WebJobs.Script","NA","NA","client","False","","","deprecated","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:577]: Do we have deprecated EOL dates, replacements, or mitigation guides for thise?

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (scottaddie) [reply _data/releases/latest/dotnet-packages.csv:577]: We don't. Many of these packages have bare READMEs and were last updated long before our team was formed.

**Pull request merged**